### ISSUE TITLE
Идея: приложение/бот для организации вечеров писем политзаключенным

### ISSUE THREAD

**Issue by loalexandra** [developers-against-repressions/devs-against-the-machine#39]
Цитируемое [сообщение](https://t-do.ru/c/1497314240/1443)

> привет! я иногда устраиваю вечера писем политзаключенным
> 
> #идея создать базу данных и на ее основе конструктор для проведения вечеров писем в поддержку политзаключенных. обычно это непросто по нескольких причинам:
> 1) списки, которые есть либо неполные, либо странные, но всегда просто табличка в Эксель, без какой-либо информации о политзаключенном (она просто не помещается). 
> 2) если участники кого-то не знают, приходится гуглить прямо во время мероприятия, а это время и силы на отсев непроверенной информации. другой вариант - ведущий должен знать биографию и дела всех, но если появилось несколько вопросов, ответить можно только на один.
> 
> Я предлагаю решение в области создания полной базы данных, с возможностью легко вывести и распечатать в читаемом виде. То есть, это про удобную базу данных с конструктором и несколько ux историй. В итоге требуется получить:
> 1) списки пзк с адресами для переписки. сейчас такие есть у группы "Сказки политзаключенным", у них просто табличка в Эксель. в распечатке она нечитаема, особенно для людей со слабым зрением.
> 2) индивидуальные карточки о пзк с фотографией, минимальной биографией, обвинениями, обстоятельствами ареста, какими организациями признан пзк.
> 3) плакаты об отдельных делах под распечатку, которые можно повесить на стены (например, Новое величие, Московское дело, Сеть и др).
> 
> Я вижу это как обновляемую базу, доступ к которой есть у нескольких активистов. Наполнение должно быть не сложнее таблички в экселе. Пользователи могут распечатать три вида документов (полный список пзк, все или отдельные карточки, все или отдельные плакаты о делах), и легко организовать вечер писем

**Comment by KichiginaTatiana** [comment]
привет! я бэкендер, могу запилить бота

**Comment by Turella** [comment]
Спасибо организаторам!
Привет @KichiginaTatiana.
Я автор идеи. Ты предлагаешь бота по биографии политзаключенных? Хорошая идея, но, думаю, без отдельных карточек под распечатку все равно не обойтись. Людям старшего возраста и широкой публике трудно с ботами.

**Comment by loalexandra** [comment]
Думаю, бот может выдавать картинки/pdf-ки для скачивания?

**Comment by KichiginaTatiana** [comment]
@Turella @loalexandra да, можно научить бота формировать pdf/excel/ещё что-нибудь и отправлять

**Comment by enshirokov** [comment]
Аналог Википедии можно сделать с полной инфой по политзекам. Движок же вроде бесплатный. Надо только домен.

**Comment by Turella** [comment]
Более или менее полной инфы много. Наша задача - упростить поддержку, причем не только онлайн, но и офлайн

**Comment by Turella** [comment]
Комменты из группы "Сказки для политзаключенных":
"прочитав о списке - сразу подумала, что это удобно для тематических встреч - например, у нас был вечер в поддержку "ингушских узников", удобно было бы сортировкой какой-то пользоваться. или - по приближающимся дням рождения. насчет вводной инфо вижу, что по ссылке речь идет о таблице - кроме нее есть вордовские файлы, не знаю, лучше ли это с точки здения выгрузки информации, но они есть. насчет актуализации: в таблице можно без прав редактирования оставлять комменты об изменениях в адресе, например (кстати это могут быть заведомо временные изменения - сизо, больница etc), но боту, вероятно, можно будет отправить сообщение, а модераторы смогут изменять инфо"
"еще идеи по наполняемости. было бы круто учитывать каким-то образом сферу интересов арестантов - вроде "о чем интересно человеку узнавать", какие жанры книжек любит, новостей на какую тему хочет. и еще, например, какие периодические издания или книжки тому или иному адресату отправлены"
+ суды, заседания

**Comment by Turella** [comment]
> @Turella @loalexandra да, можно научить бота формировать pdf/excel/ещё что-нибудь и отправлять

Я в обсуждении в чате читала, что можно вести базу данных в Телеграме. Не знаете, возможно  ли будет ее редактировать людям со средним уровнем компьютерной грамотности? Грубо говоря, которые умеют обращаться с Excel и гуглодоками, но не никогда не кодили.
